docs/api/rpc-params.md

```diff
@@ -35,12 +35,12 @@ The following identifiers are used throughout the code samples below.
 
 *At time of writing, `finality` is only available on StageNet.*
 
-To run the examples below, you will likely need a more recent block height or block hash.  You can find that here: https://staging-rpc.nearprotocol.com/status
+To run the examples below, you will likely need a more recent block height or block hash.  You can find that here: https://staging-rpc.testnet.nearprotocol.com/status
```

Comment:
These `staging-rpc.testnet.nearprotocol.com` should be `rpc.testnet.nearprotocol.com`

Reply:
I think it should be `rpc.betanet.nearprotocol.com` 
   
@potatodepaulo  pretty sure `staging-rpc.testnet.nearprotocol.com` doesn't even exist as domain

Reply:
`rpc.testnet.nearprotocol.com` does exist though. It depends on context which one we want (testing bleeding edge stuff vs just running apps on testnet).

Reply:
yep, got these changes locally already

Reply:
✅